packages/functional-tests/pages/settings/recoveryKey.ts

```diff
@@ -140,34 +128,43 @@ export class RecoveryKeyPage extends SettingsLayout {
     return this.finishButton.click();
   }
 
-  async fillOutRecoveryKeyForms(
-    password: string,
-    hint: string
-  ): Promise<string> {
+  async fillOutInfoForm(): Promise<void> {
     await expect(this.accountRecoveryKeyHeading).toBeVisible();
     await expect(this.createRecoveryKeyHeading).toBeVisible();
 
-    // View 1/4 info
     await this.getStartedButton.click();
+  }
 
+  async fillOutConfirmPasswordForm(password: string): Promise<void> {
     await expect(this.passwordHeading).toBeVisible();
 
-    // View 2/4 confirm password and generate key
     await this.createKeyPasswordTextbox.fill(password);
     await this.createKeyButton.click();
+  }
 
+  async fillOutGenerateKeyForm(): Promise<string> {
```

Comment:
methods with 'And' in the name are considered a smell usually because it indicates they don't separate a concern... Since this method isn't used outside this module, I'm actually going to remove it and incorporate back into fillOutRecoveryKeyForms/createRecoveryKey.